File: modul 1/PRAK110_2410817120001_Putri_Fatima_Azhara.py
import math

alas = 5
tinggi = 12
miring = math.sqrt(alas*alas+tinggi*tinggi)
keliling = alas+tinggi+miring
luas = 0.5*alas*tinggi

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print("")
print("Jawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {miring:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")

File: modul 1/PRAK110_2410817120001_Putri_Fatima_Azhara.c
#include <stdio.h>
#include <math.h>

int main ()
{
    int alas = 5;
    int tinggi = 12;
    int miring = sqrt(alas*alas+tinggi*tinggi);
    int keliling = alas+tinggi+miring;
    int luas = 0.5*alas*tinggi;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n",alas);
    printf("Tinggi = %d cm\n\n",tinggi);

    printf("Jawab :\n");
    printf("Sisi A = %d cm\n",tinggi);
    printf("Sisi B = %d cm\n",miring);
    printf("Sisi c = %d cm\n",alas);
    printf("Keliling = %d cm\n",keliling);
    printf("luas = %d cm\n",luas);
    return 0;
}